cli/deploy-mlnet-managed-online-endpoint.sh

```diff
@@ -0,0 +1,67 @@
+# <initialize_variables>
+BASE_PATH=endpoints/online/mlnet
+ENDPOINT_NAME=mlnet-endpoint
+DEPLOYMENT_NAME=mlnet
+ENVIRONMENT_NAME=mlnet-environment
+# </initialize_variables>
+
+# <run_image_locally_for_testing>
+docker run -d --rm -p 5000:80 --name="mlnet-test" docker.io/luisquintanilla/managed-deployment-container-mlnet:latest
```

Comment:
I agree - the fact that container exposes port 80 and yet we do local testing on port 5000 is slightly confusing. 

A bigger problem is that this sample doesn't really show any future ML.NET users how to deploy their _own_ models. We need show the Docker build steps that Luis used to generate this image. Those steps should be evident in the code folder that I sent you.

Check out this [Triton sample](https://github.com/Azure/azureml-examples/pull/490/files#diff-3690d5e03722351540dd705cfe733c5d68319f076cb029da0fe9f4b684afe864R24) for an example where I do an image build in the .sh file.